### ISSUE TITLE
Stylesheet seems not to work

### ISSUE THREAD

**Issue by DionRo** [thedevdojo/chatter#225]
Hi,

Im currently trying to make the plugin work. I thought other people would have experienced the same problem. But could not find it here. Somehow my css is not working. I have done all publishes and the files are in my public folder. Yet I have no clue why this is not working. (see screenshot).

Does anyone have the same problem and maybe even solved it? 

Laravel Framework 5.7.3
Chatter version 0.2.7
XAMPP envoirment  7.1.9

![screenshot_1](https://user-images.githubusercontent.com/26322650/45391415-1d827680-b623-11e8-9f60-df75c504d0ed.png)

Thanks for the help in advance

**Comment by far3ki** [comment]
Chapter version in your description is 0.2.7. But current version is 0.2.18. I have similar issue. Brand new clear instalation Laravel 5.7 following with 
composer require "devdojo/chatter=0.2.*"
install version 0.2.7 and not last version 0.2.18 of devdojo/chatter package. It seems, that devdojo/chatter currently not supports Laravel 5.7.

**Comment by DionRo** [comment]
Tried to reach out to them but waiting for response. Hope they can find the solution really like chatter but I do want the latest version of laravel.

Any clue on what laravel version it did work as last? 5.5?

**Comment by far3ki** [comment]
When I tested the same situation with Laravel 5.6 it worked ok, so installed version of chatter package was 0.2.18.
I would also appreciate the upgrade to Laravel 5.7

**Comment by DionRo** [comment]
Yeah would be very appreciated. But how to get in contact with the devs? I tried to send an email thats stated on their website. (Support@devdojo.com) but no idea what the estimate response time is.

**Comment by far3ki** [comment]
yeah, I have also written a post to https://devdojo.com/forums. No response yet.

**Comment by kneipp** [comment]
I had same problem. In my case missed @yield('css') and @yield('js') > app.blade.php

**Comment by kingzamzon** [comment]
run 
1. npm install
2. npm run dev
